src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int anInt = random.nextInt(5);
```

Comment:
All magic numbers in your code should be constants. You should define a constant for the number of figure types and use it in the `nextInt` call. For example, `private static final int FIGURE_COUNT = 5;` and then `random.nextInt(FIGURE_COUNT);`.